src/cmds/core/other.py

```diff
@@ -1,14 +1,31 @@
 import logging
 
-from discord import ApplicationContext, Embed, Interaction, Message, WebhookMessage, slash_command
+from discord import ApplicationContext, Embed, Interaction, Message, WebhookMessage, slash_command, ui
```

Comment:
You should probably add
```import discord```
since below you use directly from discord, but I would argue that it's better to import only whatever you need.
```suggestion
from discord import ApplicationContext, Embed, Interaction, Message, WebhookMessage, slash_command, InputTextStyle
from discord.ui import InputText, Modal
```
and use them directly below.